cmd/cluster/support/delete_test.go

```diff
@@ -0,0 +1,39 @@
+package support
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestCreateDeleteRequest(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	testCases := []struct {
+		title          string
+		clusterID      string
+		lmtSprReasonID string
+		client         *Client
+		errExpected    bool
+	}{
+		{
+			title:          "Delete Request creation",
+			clusterID:      "5a5a5a5a-5a5a-5a5a-5a5a-5a5a5a5a5a5a",
+			lmtSprReasonID: "FAFAFAFA-FAFAFAFA-FAFAFAFA-FAFAFAFA",
+			client: &Client{
+				name: "fakeSDKClient",
+			},
+			errExpected: false,
```

Comment:
Can you also create a tests that expects an error?

Reply:
Also done to be submited shortly.